### PR TITLE
SD-942 Hide outdated cards in checkout

### PR DESCRIPTION
**Pull request by KacperMekarski**

### CODE REVIEW

File: core/app/models/spree/credit_card.rb [hunk 1]
```diff
@@ -37,6 +37,11 @@ class CreditCard < Spree::Base
 
     scope :with_payment_profile, -> { where.not(gateway_customer_profile_id: nil) }
     scope :default, -> { where(default: true) }
+    scope :up_to_date, -> {
+      where('CAST(spree_credit_cards.year AS INT) > ?', DateTime.now.year).
+        or(where('CAST(spree_credit_cards.year AS INT) = ?', DateTime.now.year).
+           where('CAST(spree_credit_cards.month AS INT) >= ?', DateTime.now.month))
+    }
```
Comment (damianlegawiec) [inline core/app/models/spree/credit_card.rb:44]:
```suggestion
    scope :not_expired, -> {
      where('CAST(spree_credit_cards.year AS INT) > ?', Time.current.year).
        or(where('CAST(spree_credit_cards.year AS INT) = ?', Time.current.year).
           where('CAST(spree_credit_cards.month AS INT) >= ?', Time.current.month))
    }
```

Comment (damianlegawiec) [reply core/app/models/spree/credit_card.rb:44]: `Time.current` uses the current timezone, not UTC

**Review by damianlegawiec — COMMENTED** [review]
@KacperMekarski please make PRs to `spree/spree`

**Comment by KacperMekarski** [comment]
```
#!/bin/bash -eo pipefail
./tmp/cc-test-reporter sum-coverage tmp/codeclimate.*.json -p 4 -o tmp/codeclimate.total.json
./tmp/cc-test-reporter upload-coverage -i tmp/codeclimate.total.json

Error: response from https://api.codeclimate.com/v1/test_reports.
HTTP 401: You are not authorized for this action
Usage:
  cc-test-reporter upload-coverage [flags]
```

It seems that I don't have some privilliges, right?

**Pull request closed without merging**